lib/fog/kubevirt/compute/requests/get_service.rb

```diff
@@ -0,0 +1,16 @@
+module Fog
+  module Kubevirt
+    class Compute
+      class Real
+        def get_service(name)
+          Service.parse object_to_hash( kube_client.get_service(name, @namespace) )
```

Comment:
there shouldn't be space between the surrounding brackets

Reply:
OK